### ISSUE TITLE
[v4 Training] Issue 6: Stage 2のLossが下がらない（学習率）

### ISSUE THREAD

**Issue by tamatebox** [tamatebox/samadhi_model#7]
## 症状
- Stage 2のBCE Loss が 0.65 → 0.63 程度で停滞
- Trust scoreがClean/Drunk間で差が出ない

## 原因
HuggingFace Trainerのデフォルト学習率（5e-5）がStage 2には低すぎた。

Stage 2では Vipassana のみが学習対象で、パラメータ数が少ない（trust_head: 2→32→1）。
小さなネットワークに対して学習率が低すぎると、収束が遅くなる。

## 対処
Stage 2の学習率を1e-2に引き上げ：

```python
args = TrainingArguments(
    output_dir="./output",
    learning_rate=1e-2,  # Stage 2用に引き上げ
)
```

## 結果
- Trust score差: Clean 0.31 vs Drunk 0.13 → Clean 0.56 vs Drunk 0.07

## 教訓
- Stage毎に最適な学習率は異なる
- パラメータ数が少ないモジュールには高めの学習率が必要
- マルチステージ学習では、各ステージで`TrainingArguments`を調整することを検討

## 詳細
[docs/issues/001_training_issues_v4.md](https://github.com/tamatebox/samadhi_model/blob/main/docs/issues/001_training_issues_v4.md#issue-6-stage-2のlossが下がらない学習率)

## 関連ファイル
- `satipatthana/train/trainer.py`
